frontend/src/component/common/BreadcrumbNav/BreadcrumbNav.tsx

```diff
@@ -82,9 +87,9 @@ const BreadcrumbNav = () => {
                                     const lastItem = index === paths.length - 1;
                                     if (lastItem) {
                                         return (
-                                            <StyledParagraph key={path}>
+                                            <StyledCurrentPage key={path}>
                                                 {path}
-                                            </StyledParagraph>
+                                            </StyledCurrentPage>
```

Comment:
This could probably be simplified a bit. If the `StyledParagraph` is changed to be a `styled(Typography)`, then instead of making a new component here, you could  do:
 
```suggestion
                                            <StyledParagraph bold key={path}>
                                                 {path}
                                            </StyledParagraph>
```

Though, I'd also rename the component, because making it a Typography will (I think) default it to be a `span`, which is a more appropriate element in this case than `p`. Technically, it should also probably be a list and use `aria-current="page"` for the current page (cf the [mdn article on breadcrumbs](https://developer.mozilla.org/en-US/docs/Web/CSS/How_to/Layout_cookbook/Breadcrumb_navigation)), but that's probably fine to leave for later